x-pack/plugins/fleet/server/routes/settings/index.ts

```diff
@@ -74,11 +121,27 @@ export const registerRoutes = (router: FleetAuthzRouter, config: FleetConfigType
         fleet: { readSettings: true },
       },
       description: `Get settings`,
+      options: {
+        tags: ['Fleet internals'],
```

Comment:
the tags are actually not showing up in the `oas` api response, asked on #kibana-core